test/test.lua

```diff
@@ -296,9 +296,11 @@ function cudnntest.VolumetricConvolution()
    local outi = math.random(1,17)
    local outj = math.random(1,17)
    local outk = math.random(1,5)
-   local ini = (outi-1)*si+ki
-   local inj = (outj-1)*sj+kj
-   local ink = (outk-1)*sk+kk
+
+   local ini = outi*si+ki-1
```

Comment:
Why did you change the computation of the input sizes? Previous formulas conformed to nn input size computation.